policyengine_us/parameters/gov/usda/snap/temporary_local_benefit/dc/rate.yaml

```diff
@@ -0,0 +1,17 @@
+description: DC provides a temporary local SNAP benefit of this percentage of the maximum allotment.
+values:
+  2023-10-01: 0.1
+  # The temporary allotment expried in October 2024
```

Comment:
Based on what? I didn't see an end date in the law.

Reply:
[The DC DHS states](https://dhs.dc.gov/page/give-snap-raise-heres-what-expect):
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/73ea484b-2e69-457c-b4a5-159eca80adad)